examples/api/lib/material/page_transitions_theme/page_transitions_theme.3.dart

```diff
@@ -0,0 +1,145 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+/// Flutter code sample for [PageTransitionsTheme].
+
+void main() => runApp(const PageTransitionsThemeApp());
+
+class PageTransitionsThemeApp extends StatelessWidget {
+  const PageTransitionsThemeApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      debugShowCheckedModeBanner: false,
+      theme: ThemeData(
+        pageTransitionsTheme: PageTransitionsTheme(
+          builders: Map<TargetPlatform, PageTransitionsBuilder>.fromIterable(
+            TargetPlatform.values, value: (_) => const FadeForwardsPageTransitionsBuilder()
```

Comment:
If not specify, I think on iOS and macOS platforms, it will default to `CupertinoPageTransitionsBuilder()`, and in this example, I just wanted to show the new page transition, so I just override everything. Do you think if this is necessary?

Reply:
Then I think the comment in page_transition_theme.0.dart should be updated as the default may not always be `FadeForwardsPageTransitionsBuilder` when unspecified.